public/app/core/copy/appNotification.ts

```diff
@@ -31,12 +32,25 @@ export const createSuccessNotification = (title: string, text?: string): AppNoti
   id: Date.now(),
 });
 
-export const createErrorNotification = (title: string, text?: string): AppNotification => ({
-  ...defaultErrorNotification,
-  title: title,
-  text: text,
-  id: Date.now(),
-});
+export const createErrorNotification = (title: string, text?: any): AppNotification => {
+  // Handling if text is an error object
+  if (text && !_.isString(text)) {
+    if (text.message) {
+      text = text.message;
+    } else if (text.data && text.data.message) {
+      text = text.data.message;
+    } else {
```

Comment:
```
} else if (text.statusText) {
text = text.statusText
```

Reply:
added that and moved this to util function as I needed it in another place as well, https://github.com/grafana/grafana/blob/dashboard-react-page/public/app/core/utils/errors.ts